reference/tiflash/use-tiflash.md

```diff
@@ -118,6 +118,8 @@ explain analyze select count(*) from test.t;
 
 `cop[tiflash]` means that the task will be sent to TiFlash for processing. If you have not selected a TiFlash replica, you can try to update the statistics using the `analyze table` statement, and then check the result using the `explain analyze` statement.
 
+Note that if a table has only a single TiFlash replica and the node cannot provide service, queries in the smart mode will repeatedly retry. In this situation, you need to specify the engine or use the manual Hint to read data from TiKV.
```

Comment:
```suggestion
Note that if a table has only a single TiFlash replica and the related node cannot provide service, queries in the CBO mode will repeatedly retry. In this situation, you need to specify the engine or use the manual Hint to read data from TiKV.
```
See https://github.com/pingcap/docs-cn/pull/2883#pullrequestreview-400490430 for my suggestion.